CONTRIBUTING.md

```diff
@@ -54,6 +54,21 @@ Create an issue on the repository and provide the following information by filli
+| -------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **SEV 1**      | A critical design or functionality issue that breaks the design system, causes significant usability problems, or exposes critical security vulnerabilities. This issue impacts all users and/or essential workflows, making users unwilling or unable to complete any tasks. Requires immediate attention.                                           | Broken navigation system, complete unresponsiveness on all devices, components not rendering, inaccessible primary actions, security vulnerabilities. Accessibility: The end user is not able to access the product or complete core user tasks or activities (key navigational elements not accessible via keyboard, missing or incorrect form labels that prevent screen reader users from completing forms or actions, critical color contrast issues that prevent users from reading or interacting with essential content). |
+| **SEV 2**      | A significant problem affecting the design, usability or functionality of the system or components for a subset of users. These issues cause major disruptions and prevent users from completing tasks or workflows. Accessibility: Does not conform with WCAG 2.1 Level AA criteria and blocks core user tasks (no known workaround).                | Content that is out of view or unreachable by customers, critical buttons or links not functioning or actions that cannot be submitted. Unintentional breaking changes causing disruptions in production environments. Accessibility: Issue with a component or controller with widespread use and blocks core user tasks (no known workaround).                                                                                                                                                                                 |
```

Comment:
```suggestion
| **SEV 2**      | A significant problem affecting the design, usability or functionality of the system or components for a subset of users. These issues cause major disruptions and prevent users from completing tasks or workflows. Accessibility: Does not conform with WCAG 2.1 Level AA criteria and blocks core user tasks (no known workaround).                | Content that is out of view or unreachable by customers, critical buttons or links not functioning, or actions that cannot be submitted. Unintentional breaking changes causing disruptions in production environments. Accessibility: Issue with a component or controller with widespread use and blocks core user tasks (no known workaround).                                                                                                                                                                                 |
```

Please add newline between different categories of examples, too.

Reply:
Updated.